src/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name reason_react)
+ (wrapped false)
+ (flags :standard -bs-jsx 3)
```

Comment:
depending which melange version you install this could be changed to `(preprocessing (pps reactjs-jsx-ppx))`